qutebrowser/misc/utilcmds.py

```diff
@@ -234,3 +234,17 @@ def repeat_command(win_id, count=None):
     cmd = runners.last_command[mode_manager.mode]
     commandrunner = runners.CommandRunner(win_id)
     commandrunner.run(cmd[0], count if count is not None else cmd[1])
+
+@cmdutils.register(maxsplit=1, no_cmd_split=True, name='debug-log-capacity')
```

Comment:
Ooh okay,  am on it.
On Aug 3, 2016 11:47 PM, "Florian Bruhin" <notifications@github.com> wrote:

> In qutebrowser/misc/utilcmds.py
> <https://github.com/The-Compiler/qutebrowser/pull/1757#discussion_r73417586>
> :
>
> > @@ -234,3 +234,17 @@ def repeat_command(win_id, count=None):
> >      cmd = runners.last_command[mode_manager.mode]
> >      commandrunner = runners.CommandRunner(win_id)
> >      commandrunner.run(cmd[0], count if count is not None else cmd[1])
> > +
> > +@cmdutils.register(maxsplit=1, no_cmd_split=True, name='debug-log-capacity')
>
> No need for maxsplit=1 and no_cmd_split=True here (which control how
> arguments are splitted), but instead you should add debug=True so this
> command is only shown in the completion with --debug.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/The-Compiler/qutebrowser/pull/1757/files/4850353203f3741442907d78398ecb4739570c3a#r73417586>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AMoLNBizMd7B2dJmGcb6OBbleZ2ONMo7ks5qcP5QgaJpZM4Jb5KT>
> .
>